docs/articles/storage.md

```diff
@@ -206,6 +205,23 @@ class="sourceCode json"><code class="sourceCode json"><span id="cb1-1"><a href="
 </table>
 
 
+<a name="config-gc"></a>
+
+### Configuring garbage collection
+
+The zot configuration model allows for enabling and disabling garbage collection (`gc`) and specifying a periodic interval (`gcInterval`) for collection. 
+
+| gc      | gcInterval | Result 
+| ------- | ---------- | ------
+| false   | n/a        | GC disabled
+| omitted | n/a        | GC enabled with 1 hour interval (default)
+| true    | omitted    | GC enabled with 1 hour interval
+| true    |  0         | GC runs only once
+| true    |  >0        | GC enabled with specified interval
+
+The configuration model also allows the configuration of a tunable delay (`gcDelay`), which can be set depending on client network speeds and the size of blobs.
```

Comment:
`gcDelay` has also a default value which is 1 hour. Maybe it makes sense to specify it here?

Reply:
Yes, we should.

Reply:
Added gcDelay default in 8/14 commit